test/verify/run_verify.cpp

```diff
@@ -146,7 +147,10 @@ void run_verify::verify(const std::string& name, const migraphx::program& p) con
         for(const auto& tname : target_names)
         {
             target_info ti = get_target_info(tname);
-            auto t         = migraphx::make_target(tname);
+            // if tests disabled, skip running it
+            if(migraphx::contains(ti.disabled_tests, name))
```

Comment:
This should be done in the loop above where we add the target names, so that we dont run the ref if there is no target to run.